src/proxy/index.ts

```diff
@@ -0,0 +1,27 @@
+import { handleIssueAssigned } from "../handlers/issue-assigned";
+import { handleIssueClosed } from "../handlers/issue-closed";
+import { Context } from "../types/context";
+import { EnvConfigType } from "../types/env-type";
+import { SupportedEvents } from "../types/plugin-inputs";
+
+export interface Result {
+  status: "ok" | "failed" | "skipped";
+  content?: string;
+  reason?: string;
+}
+
+const callbacks: { [k in SupportedEvents]: (context: Context, env: EnvConfigType) => Result | Promise<Result> } = {
+  "issues.closed": handleIssueClosed,
+  "issues.assigned": handleIssueAssigned,
+  "issues.unassigned": handleIssueClosed,
```

Comment:
won't this mean that when there are 2 assignees and one gets unassigned, it will be removed from DB?

Reply:
It is true that I didn't think about that case. I will check on `unassigned` if there is still assignees